src/istgt_misc.c

```diff
@@ -243,8 +243,9 @@ poolfini(void)
 		mpools[i].endmem = NULL;
 
 	}
-	//if (gstart != NULL)
-	//	free(gstart);
+
+	if (gstart != NULL)
```

Comment:
lets not do this as memory pointed by gstart is shared among pools, even though, pools related pointers are set to NULL in poolfini.
Leak here should be ok as process is going to die now.